hazelcast/src/main/java/com/hazelcast/nio/serialization/GenericRecord.java

```diff
@@ -70,183 +71,215 @@
      *
      * @return an empty generic record builder with same class definition as this one
      */
+    @Nonnull
     Builder newBuilder();
 
     /**
      * Returned {@link Builder} can be used to have exact copy and also just to update a couple of fields. By default,
      * it will copy all the fields.
+     * So instead of following where only the `id` field is updated,
+     * <pre>
+     *     GenericRecord modifiedGenericRecord = genericRecord.createGenericRecordBuilder()
```

Comment:
I don't see any method named `createGenericRecordBuilder` ? I think it's `newBuilder`?